plugins/speculation-rules/helper.php

```diff
@@ -93,7 +93,7 @@ function plsr_get_speculation_rules() {
 		),
 	);
 
-	// Allow adding a class on any links to prevent prerendering.
+	// Allow adding a class on any links to prevent pre-rendering.
```

Comment:
I don't think this change should be made since "prerendering" is the term used. See https://developer.mozilla.org/en-US/docs/Glossary/Prerender

Reply:
I have Reverted it

Thanks @westonruter